.travis.yml

```diff
@@ -1,16 +1,27 @@
 language: go
 
+# cache:
+#   directories:
+#   - $HOME/.glide
+
 go:
 - 1.7.1
 
 branches:
   only:
   - master
   - cluster-up
+  - /^issue-[0-9]+$/
```

Comment:
Again, probably goes away. IMO there is no need to restrict branches. As I said quite often, not all issues "fix" something. I think "issue" is the more generic term. Anyways, I really don't see why we need to restrict this.